package-rfcx/rfcx/audio.py

```diff
@@ -20,7 +31,7 @@ def __local_audio_file_path(path, audio_name, audio_extension):
     return path + '/' + audio_name + "." + audio_extension
 
 def save_audio_file(destination_path, audio_id, source_audio_extension='opus'):
-    """ Prepare `url` and `local_path` and save it using function `__save_file` 
+    """ Prepare `url` and `local_path` and save it using function `__save_file` (TO BE DEPRECATED - use save_audio_file2 in future)
         Args:
             destination_path: Audio save path.
```

Comment:
I think we will just delete this.

Reply:
Then, all of v1 support for audio will be gone. Are you sure?